DEV_GUIDE.md

```diff
@@ -278,3 +278,22 @@ Finally, before committing your changes you'll want to clean up
 ```sh
 rm foo.txt .fossa.yml
 ```
+
+# MAC Developer Notes
+
+It's critically important to ensure that you have the correct JDK v8 installed before you can successfully compile the Stargate project. There are a number of versions of JDK 8 and a number of different ways to install them, but not all of them will work successfully with Stargate.
+
+Download and install Maven from this link: https://maven.apache.org/download.cgi
+
+Download the JDK from this link: https://adoptopenjdk.net/?variant=openjdk8&jvmVariant=hotspot
+
+Install that version of te JDK and then be sure to add it to your path. If you are using a newer version of the Mac OS, then you are likely using Z-Shell by default. So open your ~/.zshrc file and add the path there. Here is an example:
+
+```sh
+export JAVA_HOME="/Library/Java/JavaVirtualMachines/adoptopenjdk-8.jdk/Contents/Home"
+export MAVEN_HOME="/Users/<your user name>/development/apache-maven-3.8.1"
```

Comment:
Interesting. I've never had to set this.

Reply:
I'm not 100% sure it needs to be set. I can unset it later today and reboot to see how it affects the ability to build.

Reply:
The JAVA_HOME line is needed. However, the MAVEN_HOME line is not needed. Maven runs just fine when I don;t have that line in my .zshrc file.